common/build-style/python3-module.sh

```diff
@@ -24,15 +24,20 @@ do_build() {
 }
 
 do_check() {
-	if [ -z "$make_check_target" ]; then
-		if ! python3 setup.py --help test >/dev/null 2>&1; then
-			msg_warn "No command 'test' defined by setup.py.\n"
-			return 0
+	if python3 -m pytest --help >/dev/null 2>&1; then
+		python3 -m pytest ${make_check_args} ${make_check_target}
```

Comment:
Not sure if this is better:
```sh
PYTHONPATH="$(cd build/lib* && pwd)" python3 -m pytest ${make_check_args} ${make_check_target}
```

Reply:
Setting `PYTHONPATH` this way might improve support for testing compiled extensions, but I'm not sure it should be the default. If tests break because they can't be run with the default path (plus `.`, which is automatically added in the `python3 -m pytest` form of invocation), maybe that should be a manual override situation.